docusaurus.config.js

```diff
@@ -99,6 +99,18 @@ const config = {
         includeCurrentVersion: false,
       },
     ],
+    [
+      '@docusaurus/plugin-client-redirects',
+      {
+        redirects: [
+          // /docs/oldDoc -> /docs/newDoc
```

Comment:
@ojbravo Could you remove the comment or re-write the comment to be our paths like `/documentation/oldDo` -> `/documentation/newDoc`

Reply:
Update made. Comment removed.